scene/gui/control.cpp

```diff
@@ -2063,7 +2063,7 @@ Control *Control::find_next_valid_focus() const {
 		// If the focus property is manually overwritten, attempt to use it.
 
 		if (!data.focus_next.is_empty()) {
-			Node *n = get_node(data.focus_next);
+			Node *n = get_node_or_null(data.focus_next);
```

Comment:
One of the cases where I'm unsure if an error should be made here, will add explicit error if desired

Reply:
Probably should have an error message, since it's a user configurable node path.

Reply:
Then I'll add one for those, instead of the null return